store/items.ts

```diff
@@ -0,0 +1,63 @@
+import Vue from 'vue';
+import { BaseItemDto } from '@jellyfin/client-axios';
+import { ActionTree, GetterTree, MutationTree } from 'vuex';
+import reduce from 'lodash/reduce';
+
+export interface ItemsState {
+  items: Record<string, BaseItemDto>;
+  ids: string[];
+}
+
+export const defaultState = (): ItemsState => ({
+  items: {},
+  ids: []
```

Comment:
```suggestion
  byId: {},
  allIds: []
```

This removes some repetition in the access and is more explicit about what is what.

Instead of doing `$store.state.items.items` you'd do `$store.state.items.byId`.

Reply:
Though it's only `$store.state.items` or `$store.state.ids` currently, there's not an `items` object root :)

Reply:
When you access `$store.state` in the app, you get the RootState, which is this:
![image](https://user-images.githubusercontent.com/19396809/109711800-95500d00-7b9f-11eb-8a3c-5bf812b0d149.png)

So you'll have `state.items.items` and `state.items.ids` currently.

Reply:
Noted